build_dlkit.py

```diff
@@ -876,6 +876,34 @@ def update_hash_file(self):
             }
             json.dump(build_file, build_file_handle)
 
+    def junit(self, create_parent_dir=False):
```

Comment:
@birdland It seems misleading to include these Java commands in the builder, but not have the builder files...